src/components/ha-color-picker.ts

```diff
@@ -32,7 +33,16 @@ export class HaColorPicker extends LitElement {
 
   @property({ type: Boolean }) public disabled = false;
 
-  _valueSelected(ev) {
+  @query("ha-select") private _select!: HaSelect;
+
+  connectedCallback(): void {
+    super.connectedCallback();
+    this._select.layoutOptions();
```

Comment:
Why do we have to do this on connected?

Reply:
When going back from yaml editor to UI editor, the displayed value is still the old one but the field wasn't updated when unmounted.

Reply:
Is the UI editor cached?

Can we do it on value change then and the value is custom?